Cargo.toml

```diff
@@ -26,11 +26,16 @@ clock = ["std", "winapi", "iana-time-zone", "android-tzdata"]
 wasmbind = ["wasm-bindgen", "js-sys"]
 unstable-locales = ["pure-rust-locales"]
 __internal_bench = []
+rkyv     = ["rkyv_32"]
```

Comment:
Nits: would prefer to elide the fancy formatting with extra spaces and use `-` instead of `_` for chrono's features. Please rename `_ck` to `-validation`.

Reply:
Fixed